src/shared/bus-unit-util.c

```diff
@@ -407,11 +407,15 @@ static int bus_append_cgroup_property(sd_bus_message *m, const char *field, cons
 
         if (streq(field, "Delegate")) {
 
-                r = parse_boolean(eq);
-                if (r < 0)
-                        return bus_append_strv(m, "DelegateControllers", eq, EXTRACT_QUOTES);
+                if (isempty(eq))
+                        r = sd_bus_message_append(m, "(sv)", "Delegate", "b", 0);
```

Comment:
The value `0` may be `1`. See ff1b8455c26b560641d476b426209e297209333a and #7336.